### DISCUSSION TITLE
Benchmarks elderjs vs other SSGs

### DISCUSSION THREAD

**Discussion by s-kris** [Elderjs/elderjs#166] [General]
Elderjs always had the overhead of compiling svelte but the generation of pages is blazing fast. 
I did some benchmarks to put hugo, zola, eleventy, and plenti to test. Without any surprise, zola was crushing it.
All other SSGs were faster than elderjs in orders of magnitude when building <100 pages. But when building 10,000 pages, elderjs even beat hugo standing next to zola. That's absolutely insane. Kudos to @nickreese 

![E0QWtG6VgAAQSlM](https://user-images.githubusercontent.com/3975305/116805176-32003e00-ab42-11eb-9d79-2acee8d25274.png)

You can read more thoughts here: https://twitter.com/_skris/status/1388259901080621056?s=20

**Comment by nickreese** [comment]
@s-kris awesome! Thanks for doing this. Awesome to see it performed so well. There is probably a 30% speed increase possible but at this point more CPU cores are cheaper than dev time for my needs.

**Reply by s-kris** [reply]
Absolutely. I can understand. :)
